script/test/fixtures/v3/docker-compose-full-example.yaml

```diff
@@ -39,7 +39,7 @@ services:
           cpus: '0.0001'
           memory: 20M
       restart_policy:
-        condition: on_failure
+        condition: on-failure
```

Comment:
unfortunatley, docker says that it works with either _ or -. :-1: but keep this change in!